critiquebrainz/frontend/templates/review/revision_results.html

```diff
@@ -1,6 +1,6 @@
 {% for result in results %}
   <tr>
-    <td><a href="{{ url_for('review.entity', id=review.id, rev=result[0]+1) }}">Revision {{ result[0] + 1 }}</a></td>
+    <td><a href="{{ url_for('review.entity', id=review.id, rev=result[0]+1) }}">{{ _('Revision') }} {{ result[0] + 1 }}</a></td>
```

Comment:
It's probably better to make it more explicit:

    {{ _('Revision %(number)s', number=result[0]+1) }}